src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,57 @@
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  position: relative;
```

Comment:
Pay attention to CSS properties order
https://9elements.com/css-rule-order/

Reply:
I agree with you. I will fix it